distribution/tools/upgrade-cli/build.gradle

```
@@ -21,7 +21,10 @@ dependencies {
   implementation "com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}"
   testImplementation project(":test:framework")
   testImplementation 'com.google.jimfs:jimfs:1.2'
-  testRuntimeOnly 'com.google.guava:guava:32.0.1-jre'
+  testRuntimeOnly 'com.google.guava:guava:32.1.1-jre'
```

@owaiskazi19 could you please try 
```suggestion
testRuntimeOnly('com.google.guava:guava:32.1.0-jre') {
    transitive = false
  }
```

---

This is great! Thanks @reta. Precommit passed on my local. Let's wait for the green check here.